roles/keystone/tasks/main.yml

```diff
@@ -31,6 +31,15 @@
   notify:
     - restart keystone services
 
+- name: install ldap pip reqs in package venv
```

Comment:
This is not bug in Keystone as it is not a requirement... It is a *possible* requirement, based on your configuration. Same idea when it comes to things like MySQL vs postgres... It is implementation-specific. We can add these to the giftwrap build.